backstop/tests/tag.html

```diff
@@ -109,13 +109,13 @@ <h2>Default tag containers</h2>
           tabindex="0"
         >
           <span
-            class="iui-label"
+            class="iui-tag-label"
             title="Tag"
           >
             Tag
           </span>
           <button
-            class="iui-button iui-borderless iui-small"
+            class="iui-tag-button"
             aria-label="Delete tag"
             type="button"
             tabindex="-1"
```

Comment:
Why it is set to `-1`? It is not tabbable because of that.

Reply:
No idea.. 
I also see we have `tabindex` on whole tag set in html.. in react we do not, so in react only close button is focusable.
@FlyersPh9 do you know the intention here?

Reply:
I think the reasoning was clicking anywhere on the tag would delete it… but thinking about that now, I'm not sure I like that interaction.  It is not ideal that you have to hit the tab key twice to get to the 2nd tag (1st tab goes to the tag, 2nd tab goes to delete button).

Reply:
I do agree if we let focus whole tab, something should be interactable, though :/
So idk, Im open to doing what ever you guys feel would be the best.

Reply:
But how close icon should be clicked then? I see in react we allow to focus close icon.

Reply:
Should we get rid of focus on tag and just have close button focusable?

Reply:
I'm fine with that.

Reply:
Ok, so I have removed focus from tag and its styling..
Later we can revisit, cause I see mui has whole tag focusable and interactive, not button inside it (but then we should have the button without separate styling, IMO)